plugins/DuskVerb/tools/tuner/apply_best_and_evaluate.py

```
@@ -82,6 +82,7 @@ def peak_aligned_thirdoct(p, t0=0.05, t1=1.0):
 
 def report(preset, params, vst3, anchor):
     dv_dir = Path("/tmp/eval_best")
```

Ruff (0.15.18)

[error] Probable insecure usage of temporary file or directory: "/tmp/eval_best"

(S108)